kinto/plugins/history/views.py

```diff
@@ -43,3 +45,88 @@ def _extract_filters(self, queryparams=None):
             filters_str_id.append(filt)
 
         return filters_str_id
+
+
+class Revert(resource.UserResource):
+
+    def __init__(self, request):
+        self.request = request
+        self.bucket_id = request.matchdict['bucket_id']
+        if 'since' not in request.params:
+            raise_invalid(self.request, 'params', 'since', 'since request parameter must be provided')
+        self.since = int(request.params['since'])
+        filters = super(Revert, self)._extract_filters(request.params)
```

Comment:
If you inherit from `UserResource` just to access that private method, then we should refactor that. We could have a `extract_filters` function in `kinto.core.resource` instead, that both places would call